e2e/vue-start/basic-vue-query/src/routes/posts.$postId.tsx

```
@@ -3,7 +3,7 @@ import {
   ErrorComponent,
   Link,
   createFileRoute,
-  type ErrorComponentProps,
+  ErrorComponentProps,
```

ESLint

[error] Member 'ErrorComponentProps' of the import declaration should be sorted alphabetically.

(sort-imports)